src/lib/types/ZodSchemas.ts

```diff
@@ -1,69 +1,112 @@
 import { z } from 'zod';
 import { ISOStringToSQLTimestamp } from 'lib/units/convertTimestamp';
-import { endOfToday, isValid } from 'date-fns';
+import { isValid } from 'date-fns';
 
-/**
- * schema for parsing location information,
- * geo-information is validated to be in range
- **/
-export const zLocation = z.object({
-  lat: z.optional(z.string()  // input is a string which has to be transformed
-    .transform(str => Number(str))
-    .refine((num) => num >= -90, 'should be greater than or equal to -90')
-    .refine((num) => num <= 90, 'should be less than or equal to 90')
-    .or(z.number().gte(-90).lte(90)) // no need to transform if input is already a number (e.g. when coming from req. body)
-  ),
-  long: z.optional(z.string()
-    .transform(str => Number(str))
-    .refine((num) => num >= -180, 'should be greater than or equal to -180')
-    .refine((num) => num <= 180, 'should be less than or equal to 180')
-    .or(z.number().gte(-180).lte(180)) // no need to transform if input is already a number (e.g. when coming from req. body)
-  ),
-  rad: z.string().optional().default('200')
-    .transform(str => Number(str))
-    .refine((num) => num > 0, 'should be positive')
-    .or(z.number().positive())  // no need to transform if input is already a number (e.g. when coming from req. body)
-  ,
-  location_name: z.string().optional(),
-});
+
+/* GENERAL SCHEMAS */
+/* used when parsing an id from query param */
+export const zIdFromString = z.string()
+  .transform(str => parseInt(str, 10))
+  .refine((num) => num > 0, 'id has to be positive');
+
+/* used when parsing an id from body param */
+export const zId = z.number().int().positive();
 
 /**
  * schema for parsing a valid time range
  * must be a valid ISO date and defaults to 2022-01-01 -> now
  **/
 export const zTime = z.object({
   /* FIXME: Sort out proper ranges later */
-  start_date: z.string().default('2022Z' /* new year 2022 */)
+  startTime: z.string().default('2022Z' /* new year 2022 */)
     .refine(str => isValid(new Date(str)), 'Unable to parse string as Date')
     //.refine(str => new Date(str) >= new Date('2022Z'), 'must be after 2022')
     .transform(str => new Date(str).toISOString()),
-  end_date: z.string().default(new Date().toISOString() /* current time */)
+  endTime: z.string().default(new Date().toISOString() /* current time */)
     .refine(str => new Date(str).getTime() > 0, 'Unable to parse string as Date')
-    .refine(str => new Date(str) <= endOfToday(), "can't be a future date")
+    //.refine(str => new Date(str) <= endOfToday(), "can't be a future date")
```

Comment:
re-enable the boundaries of lines 23 and 27 now that the image goes live?
or perhaps it should stay until final release?